### ISSUE TITLE
Going straight to certain ember data urls result in an error

### ISSUE THREAD

**Issue by toddjordan** [ember-learn/ember-api-docs#250]
Reproduce: go straight https://emberjs.com/api-beta/ember-data/2.14.3/classes/DS.ManyArray instead of navigating to it from the app.

it works from the client when ember is already hydrated, but fails when going through fastboot with:

```
Error while processing route: project-version.classes.class.index Cannot read property 'missing' of undefined TypeError: Cannot read property 'missing' of undefined
    at Class.getRevId (/Users/jordanto/dev/ember-api-docs/tmp/funnel-output_path-CtStZS89.tmp/assets/ember-api-docs/services/meta-store.js:27:1)
    at Class._callee$ (/Users/jordanto/dev/ember-api-docs/tmp/funnel-output_path-CtStZS89.tmp/assets/ember-api-docs/adapters/application.js:121:1)
    at tryCatch (/Users/jordanto/dev/ember-api-docs/tmp/funnel-output_path-CtStZS89.tmp/assets/vendor/regenerator-runtime/runtime.js:63:1)
    at GeneratorFunctionPrototype.invoke [as _invoke] (/Users/jordanto/dev/ember-api-docs/tmp/funnel-output_path-CtStZS89.tmp/assets/vendor/regenerator-runtime/runtime.js:337:1)
    at GeneratorFunctionPrototype.prototype.(anonymous function) [as next] (/Users/jordanto/dev/ember-api-docs/tmp/funnel-output_path-CtStZS89.tmp/assets/vendor/regenerator-runtime/runtime.js:96:1)
    at step (/Users/jordanto/dev/ember-api-docs/tmp/funnel-output_path-CtStZS89.tmp/assets/ember-api-docs/adapters/application.js:52:1)
    at /Users/jordanto/dev/ember-api-docs/tmp/funnel-output_path-CtStZS89.tmp/assets/ember-api-docs/adapters/application.js:70:1
    at initializePromise (/Users/jordanto/dev/ember-api-docs/tmp/funnel-output_path-CtStZS89.tmp/assets/vendor/ember/ember.debug.js:50230:1)
    at new Promise (/Users/jordanto/dev/ember-api-docs/tmp/funnel-output_path-CtStZS89.tmp/assets/vendor/ember/ember.debug.js:50718:1)
    at Class.<anonymous> (/Users/jordanto/dev/ember-api-docs/tmp/funnel-output_path-CtStZS89.tmp/assets/ember-api-docs/adapters/application.js:49:1)
    at Class.findRecord (/Users/jordanto/dev/ember-api-docs/tmp/funnel-output_path-CtStZS89.tmp/assets/ember-api-docs/adapters/application.js:171:1)
    at Class.superWrapper [as findRecord] (/Users/jordanto/dev/ember-api-docs/tmp/funnel-output_path-CtStZS89.tmp/assets/vendor/ember/ember.debug.js:42834:1)
    at _find (/Users/jordanto/dev/ember-api-docs/tmp/funnel-output_path-CtStZS89.tmp/assets/addon-tree-output/modules/ember-data/-private.js:8526:1)
    at Class._fetchRecord (/Users/jordanto/dev/ember-api-docs/tmp/funnel-output_path-CtStZS89.tmp/assets/addon-tree-output/modules/ember-data/-private.js:10509:1)
    at _fetchRecord (/Users/jordanto/dev/ember-api-docs/tmp/funnel-output_path-CtStZS89.tmp/assets/addon-tree-output/modules/ember-data/-private.js:10570:1)
    at Class._flushPendingFetchForType (/Users/jordanto/dev/ember-api-docs/tmp/funnel-output_path-CtStZS89.tmp/assets/addon-tree-output/modules/ember-data/-private.js:10669:1)
    at cb (/Users/jordanto/dev/ember-api-docs/tmp/funnel-output_path-CtStZS89.tmp/assets/vendor/ember/ember.debug.js:21389:1)
    at OrderedSet.forEach (/Users/jordanto/dev/ember-api-docs/tmp/funnel-output_path-CtStZS89.tmp/assets/vendor/ember/ember.debug.js:21194:1)
    at MapWithDefault.forEach (/Users/jordanto/dev/ember-api-docs/tmp/funnel-output_path-CtStZS89.tmp/assets/vendor/ember/ember.debug.js:21397:1)
    at Class.flushAllPendingFetches (/Users/jordanto/dev/ember-api-docs/tmp/funnel-output_path-CtStZS89.tmp/assets/addon-tree-output/modules/ember-data/-private.js:10551:1)
    at invokeWithOnError (/Users/jordanto/dev/ember-api-docs/tmp/funnel-output_path-CtStZS89.tmp/assets/vendor/ember/ember.debug.js:2197:1)
    at Object.flush (/Users/jordanto/dev/ember-api-docs/tmp/funnel-output_path-CtStZS89.tmp/assets/vendor/ember/ember.debug.js:2256:1)
    at Object.flush (/Users/jordanto/dev/ember-api-docs/tmp/funnel-output_path-CtStZS89.tmp/assets/vendor/ember/ember.debug.js:2380:1)
    at Object.end (/Users/jordanto/dev/ember-api-docs/tmp/funnel-output_path-CtStZS89.tmp/assets/vendor/ember/ember.debug.js:2450:1)
    at Timeout._onTimeout (/Users/jordanto/dev/ember-api-docs/tmp/funnel-output_path-CtStZS89.tmp/assets/vendor/ember/ember.debug.js:3016:1)
    at ontimeout (timers.js:365:14)
    at tryOnTimeout (timers.js:237:5)
    at Timer.listOnTimeout (timers.js:207:5)
TypeError: Cannot read property 'missing' of undefined
    at Class.getRevId (/Users/jordanto/dev/ember-api-docs/tmp/funnel-output_path-CtStZS89.tmp/assets/ember-api-docs/services/meta-store.js:27:1)
    at Class._callee$ (/Users/jordanto/dev/ember-api-docs/tmp/funnel-output_path-CtStZS89.tmp/assets/ember-api-docs/adapters/application.js:121:1)
    at tryCatch (/Users/jordanto/dev/ember-api-docs/tmp/funnel-output_path-CtStZS89.tmp/assets/vendor/regenerator-runtime/runtime.js:63:1)
    at GeneratorFunctionPrototype.invoke [as _invoke] (/Users/jordanto/dev/ember-api-docs/tmp/funnel-output_path-CtStZS89.tmp/assets/vendor/regenerator-runtime/runtime.js:337:1)
    at GeneratorFunctionPrototype.prototype.(anonymous function) [as next] (/Users/jordanto/dev/ember-api-docs/tmp/funnel-output_path-CtStZS89.tmp/assets/vendor/regenerator-runtime/runtime.js:96:1)
    at step (/Users/jordanto/dev/ember-api-docs/tmp/funnel-output_path-CtStZS89.tmp/assets/ember-api-docs/adapters/application.js:52:1)
    at /Users/jordanto/dev/ember-api-docs/tmp/funnel-output_path-CtStZS89.tmp/assets/ember-api-docs/adapters/application.js:70:1
    at initializePromise (/Users/jordanto/dev/ember-api-docs/tmp/funnel-output_path-CtStZS89.tmp/assets/vendor/ember/ember.debug.js:50230:1)
    at new Promise (/Users/jordanto/dev/ember-api-docs/tmp/funnel-output_path-CtStZS89.tmp/assets/vendor/ember/ember.debug.js:50718:1)
    at Class.<anonymous> (/Users/jordanto/dev/ember-api-docs/tmp/funnel-output_path-CtStZS89.tmp/assets/ember-api-docs/adapters/application.js:49:1)
    at Class.findRecord (/Users/jordanto/dev/ember-api-docs/tmp/funnel-output_path-CtStZS89.tmp/assets/ember-api-docs/adapters/application.js:171:1)
    at Class.superWrapper [as findRecord] (/Users/jordanto/dev/ember-api-docs/tmp/funnel-output_path-CtStZS89.tmp/assets/vendor/ember/ember.debug.js:42834:1)
    at _find (/Users/jordanto/dev/ember-api-docs/tmp/funnel-output_path-CtStZS89.tmp/assets/addon-tree-output/modules/ember-data/-private.js:8526:1)
    at Class._fetchRecord (/Users/jordanto/dev/ember-api-docs/tmp/funnel-output_path-CtStZS89.tmp/assets/addon-tree-output/modules/ember-data/-private.js:10509:1)
    at _fetchRecord (/Users/jordanto/dev/ember-api-docs/tmp/funnel-output_path-CtStZS89.tmp/assets/addon-tree-output/modules/ember-data/-private.js:10570:1)
    at Class._flushPendingFetchForType (/Users/jordanto/dev/ember-api-docs/tmp/funnel-output_path-CtStZS89.tmp/assets/addon-tree-output/modules/ember-data/-private.js:10669:1)
    at cb (/Users/jordanto/dev/ember-api-docs/tmp/funnel-output_path-CtStZS89.tmp/assets/vendor/ember/ember.debug.js:21389:1)
    at OrderedSet.forEach (/Users/jordanto/dev/ember-api-docs/tmp/funnel-output_path-CtStZS89.tmp/assets/vendor/ember/ember.debug.js:21194:1)
    at MapWithDefault.forEach (/Users/jordanto/dev/ember-api-docs/tmp/funnel-output_path-CtStZS89.tmp/assets/vendor/ember/ember.debug.js:21397:1)
    at Class.flushAllPendingFetches (/Users/jordanto/dev/ember-api-docs/tmp/funnel-output_path-CtStZS89.tmp/assets/addon-tree-output/modules/ember-data/-private.js:10551:1)
    at invokeWithOnError (/Users/jordanto/dev/ember-api-docs/tmp/funnel-output_path-CtStZS89.tmp/assets/vendor/ember/ember.debug.js:2197:1)
    at Object.flush (/Users/jordanto/dev/ember-api-docs/tmp/funnel-output_path-CtStZS89.tmp/assets/vendor/ember/ember.debug.js:2256:1)
    at Object.flush (/Users/jordanto/dev/ember-api-docs/tmp/funnel-output_path-CtStZS89.tmp/assets/vendor/ember/ember.debug.js:2380:1)
    at Object.end (/Users/jordanto/dev/ember-api-docs/tmp/funnel-output_path-CtStZS89.tmp/assets/vendor/ember/ember.debug.js:2450:1)
    at Timeout._onTimeout (/Users/jordanto/dev/ember-api-docs/tmp/funnel-output_path-CtStZS89.tmp/assets/vendor/ember/ember.debug.js:3016:1)
    at ontimeout (timers.js:365:14)
    at tryOnTimeout (timers.js:237:5)
    at Timer.listOnTimeout (timers.js:207:5)
```

**Comment by MartinMalinda** [comment]
I am not 100% sure, but I think this should be fixed by https://github.com/ember-learn/ember-api-docs/pull/248/files#diff-a98966b4505d2ca5536fea95f854f861R34

**Comment by toddjordan** [comment]
possibly fixed by #243 will do some testing once the fixed issue is deployed and close this as appropriate.

**Comment by toddjordan** [comment]
Fixed thanks to @MartinMalinda 🎉
